conformance/tests/ot_bulk.py

```diff
@@ -34,3 +37,30 @@ def test_bulkload(O):
             (res["count"], 6))
 
     return errors
+
+
+def test_bulkload_validate(O):
```

Comment:
Would it be worth adding in a unit test that actually tests the batching? IE, lots of vertices or elements with large amounts of attached data?